setup.py

```diff
@@ -26,6 +26,13 @@ def get_version():
     long_description=open('README.rst').read(),
     url='https://github.com/scrapinghub/scrapy-autoextract',
     packages=find_packages(),
+    install_requires=[
+        # FIXME: uncomment after release
+        # 'autoextract-poet',
+        'scrapinghub-autoextract==0.3.0',
+        'scrapy',
```

Comment:
scrapy was not in install_requires on purpose; usually plugins don't declare their frameworks as dependencies. In past it was more important, because it could have triggered unintended upgrades of the framework (e.g. `pip install -U scrapy-autoextract` could have upgraded Scrapy if Scrapy is in install_requires). Now I think pip defaults not to do this, but still, I think it is safer not to have it here.

Reply:
It's complicated. I'm removing the library from the project's dependencies but I personally believe it should be listed as a requirement because our library makes use of its code. Scrapy and our library are interdependent.